stdeb.cfg

```diff
@@ -1,5 +1,5 @@
 [DEFAULT]
-Depends3: python3-setuptools
+Depends3: python3-setuptools, python3-importlib-metadata
 Conflicts3: python-osrf-pycommon
 Suite3: xenial yakkety zesty artful bionic cosmic disco eoan focal stretch buster
```

Comment:
@clalancette this is the root of the conflict, we have `python3-importlib-metadata` as a dependency _and_ `xenial`/`bionic` (plus more) suites that don't have that deb.

So to take this we'd have to drop those suites, which then means we couldn't make bugfixes to those other suites. Unless we set up a separate major version for that I guess.

Reply:
OK, yeah.  To be able to still release for 16.04 and 18.04, we'd need a separate branch and major version.  It's not ideal, but it seems like the best way forward.  @wjwwood Are you on board with doing that?  Are there other pitfalls with branching off here?

Reply:
I'm not sure we've ever done that for a python package before. I would be concerned that running `ros_release_python` on two different versions of this package might cause issues, but I don't know that.

Reply:
@nuclearsandwich have you ever run into this case (having different `stdeb.cfg` files for different distributions (suites)?

Reply:
As long as the suites described (both `Suite` and `Suite3` values) are mutually distinct and the version numbering does not conflict for source tarballs  it _should_ work. I can set up a proof-of-concept if needed or we can schedule the first split releases during a time when I'm on deck to handle any unforeseen consequences.